metadatastore/conf.py

```diff
@@ -3,18 +3,29 @@
 import logging
 
 logger = logging.getLogger(__name__)
-filename = os.path.join(os.path.expanduser('~'), '.config', 'metadatastore',
-                        'connection.yml')
-if os.path.isfile(filename):
-    with open(filename) as f:
-        connection_config = yaml.load(f)
-    logger.debug("Using db connection specified in config file. \n%r",
-                 connection_config)
-else:
-    connection_config = {
-        'database': 'test',
-        'host': "localhost",
-        'port': 27017,
-        }
```

Comment:
Yes, Stuart wants MDS and FS to raise rather than risk putting real data in a "default" location that is not being intentionally maintained. In practice, there will still be a default, but it will come from our conda packaging, not from MDS and FS themselves.